cmd/drone-bazelisk-ecr/plugin.go

```diff
@@ -16,17 +16,18 @@ import (
 
 // plugin configuraion
 type plugin struct {
-	Target           string `required:"true"`
-	Registry         string `required:"true"`
-	CreateRepository bool   `split_words:"true"`
-	Repository       string
-	Tag              string
-	AccessKey        string `split_words:"true"`
-	SecretKey        string `split_words:"true"`
-	Bazelrc          string
-	Command          string
-	CommandArgs      string `split_words:"true"`
-	TargetArgs       string `split_words:"true"`
+	Target             string `required:"true"`
+	Registry           string `required:"true"`
+	CreateRepository   bool   `split_words:"true"`
+	Repository         string
+	Tag                string
+	AccessKey          string `split_words:"true"`
+	SecretKey          string `split_words:"true"`
+	Bazelrc            string
+	Command            string
+	CommandArgs        string `split_words:"true"`
+	EngflowBesKeywords bool
```

Comment:
This will ensure it reads in the env var `PLUGIN_ENGFLOW_BES_KEYWORDS`, as opposed to `PLUGIN_ENGFLOWBESKEYWORDS`, which translates to the following drone.yml input:
```
settings:
  engflow_bes_keywords: true
```

You can update [this unit test](https://github.com/kanopy-platform/drone-bazelisk-ecr/blob/6ad40f98ac4b29ed107e697b26ce6ca1ecc84cf3/cmd/drone-bazelisk-ecr/plugin_test.go#L83) to validate this behavior.

```suggestion
	EngflowBesKeywords bool `split_words:"true"`
```

Reply:
Oh that is definitely what I want. I couldn't figure out why the heck the other boolean had split word on it, for the strings I figured it was doing word splitting on the values or something, so multiple command arguments would work right for instance.

This makes more sense :)